assets/src/edit-story/elements/text/edit.js

```diff
@@ -199,51 +188,11 @@ function TextEdit({
     evt.stopPropagation();
   };
 
-  // Finally update content for element on unmount.
-  useEffect(
-    () => () => {
-      const newState = lastKnownState.current;
-      const newHeight = editorHeightRef.current;
-      wrapperRef.current.style.height = '';
-      if (newState) {
-        // Remember to trim any trailing non-breaking space.
-        const properties = {
-          content: stateToHTML(lastKnownState.current, {
```

Comment:
Hmm, shouldn't we still update the content? Is the content changed without it at all? Did you test it works as expected?

Reply:
hmm... Yes, we need it. Just focused on the padding and size issue without textupdate. Maybe need to findout the different way to update the size and padding. Thank you!